core/dbt/adapters/base/impl.py

```diff
@@ -1298,6 +1301,57 @@ def render_column_constraint(cls, constraint: ColumnLevelConstraint) -> str:
         else:
             return ""
 
+    @available
+    def render_ddl(self, columns: Dict[str, Dict]) -> List:
```

Comment:
this name feels too generic - a lot of what dbt does is render ddl. how about `render_column_constraint_ddl`?